proxy/src/coco/config.rs

```diff
@@ -57,6 +64,7 @@ pub fn default(
     key: keys::SecretKey,
     path: impl AsRef<std::path::Path>,
 ) -> Result<net::peer::PeerConfig<Disco, keys::SecretKey>, error::Error> {
+    env::set_var(RAD_HOME, path.as_ref());
```

Comment:
Note that this will only set the env of this process and its children. Is that the intention?

Reply:
Actually, I think this is useless to me. I thought we needed it for the rad remote helper.

Reply:
That would make sense, if we're spawning it as a child